alf/utils/dist_utils.py

```diff
@@ -18,10 +18,31 @@
 import numpy as np
 import torch
 import torch.distributions as td
+from torch.distributions import constraints
 import torch.nn as nn
 
 import alf.nest as nest
 from alf.tensor_specs import TensorSpec
+from alf.utils import common
+
+
+def get_invertable(cls):
+    """A helper function to make the transformation invertable by turning
```

Comment:
What exactly does this mean?

Reply:
Some transformations may not be able to provide accurate inversion therefore the difference between x and g_inv(g(x)) is large. This is could lead to unstable training in practice.
Documentation is updated.

Reply:
I mean how does the cache work? One or two sentences can be added to the documentation.

Reply:
Now add more explanations on how the cache work.